scripts/data_utils.py

```diff
@@ -632,7 +636,66 @@ def unmask_urls(text, url_dict={}):
     if total_size > 0:
         yield current_chunk, total_size
 
+def get_payload_and_headers_cohere(
+    text, aad_token) -> Tuple[Dict, Dict]:
+    oai_headers =  {
+        "Content-Type": "application/json",
+        "Authorization": f"Bearer {aad_token}",
+    }
+
+    cohere_body = { "texts": [text], "input_type": "search_document" }
+    return cohere_body, oai_headers
+
+def allowSelfSignedHttps(allowed):

```

Comment:
why is this needed?

Reply:
This is not needed. Please delete the legacy code. @SophieGarden